src/TLuaInterpreter.cpp

```diff
@@ -8689,10 +8708,10 @@ int TLuaInterpreter::setAreaUserData(lua_State* L)
         if (!isWarningIssued && host.mpMap->mDefaultVersion <= 16 && host.mpMap->mSaveVersion < 17) {
             QString warnMsg = tr("[ WARN ]  - Lua command setAreaUserData() used - it is currently flagged as experimental!");
             QString infoMsg = tr("[ INFO ]  - To be fully functional the above command requests a revision to the map file format\n"
-                                         "and although that has been coded it is NOT enabled so this feature's effects\n"
```

Comment:
ℹ️ I try and line the second and later lines up so that the text starts with the first non-whitespace after the `-` and then try to fit the lines into an 80 character max (I don't always succeed)...! Trouble is I tend to forget the `clang-format: on/off` comments that prevent them from being "tidied up afterwards. 😊

Reply:
I think we can deal with this and tweak clang-format settings later if needed :wink: